resources/views/landing/features.blade.php

```diff
@@ -1,36 +1,34 @@
-<a name="features"></a>
-<section class="features">
-    <div class="list">
-        <p class="feature">
-            <i class="icon fa fa-arrows-alt"></i><br>
-            Responsive out of the box
-        </p>
-        <p class="feature">
-            <i class="icon fa fa-clock-o"></i><br>
-            Easy and fast
-        </p>
-        <p class="feature">
-            <i class="icon fa fa-github"></i><br>
-            Powered by GitHub
-        </p>
+<div class="bg-white w-full pt-8">
+    <div class="container mx-auto text-center pt-4 pb-2">
+        <h2 class="mt-6 text-blue-darkest">Everything you need</h2>
+        <div class="title-underline"></div>
+        <p class="md:w-4/5 mx-auto text-grey leading-normal text-base md:px-8 my-6">In hac habitasse platea dictumst. Vivamus adipiscing fermentum quam voutpat aliquam. Integer et elit eget elit facilisis tristique. Nam vel iaculis mauirs.</p>
     </div>
-    <div class="screenshot">
-        <a href="https://gistlog.co/mattstauffer/1c76d40371b295184845">
-            <img src="{{ asset('img/features-screenshot.png' )}}" alt="Screenshot of a typical Gistlog post">
-        </a>
+</div>
+
+<div class="flex flex-col sm:flex-row container mx-auto leading-normal pb-8 mt-8 pt-4">
+    <div class="flex-1 text-center  mb-8">
+        <figure class="lg:px-4 sm:mx-8  min-h-6 w-32 sm:w-auto mx-auto">
+            <img src="{{ asset('img/icon-responsive.svg') }}" alt="Responsive design - {{ config('app.name') }}">
```

Comment:
Let's be consistent; some times with `asset('/img`, sometimes `asset(img`.

I prefer it without the leading slash.